index.js

```diff
@@ -1,31 +1,48 @@
 /* jshint node:true */
 'use strict';
 
-var getHoodieServer = require('hoodie');
+var Hapi = require('hapi');
+var hoodie = require('hoodie');
 var url = require('url');
 var path = require('path');
 var proxy = require('http-proxy-middleware');
 
 var mergeTrees = require('broccoli-merge-trees');
 var Funnel = require('broccoli-funnel');
 
-var hoodie_options = {
-  port: 4201
+var hapiOptions = {
+  server: {},
+  connection: {
+    port: 4201
+  }
+};
+var hoodieOptions = {
+  paths: {
+    public: 'dist'
+  }
 };
 
 function hoodieMiddleware(config) {
-  config.app.use('/hoodie', proxy({target: 'http://localhost:' + hoodie_options.port}));
+  config.app.use('/hoodie', proxy({target: 'http://localhost:' + hapiOptions.connection.port}));
+
+  var server = new Hapi.Server(hapiOptions.server);
+  server.connection(hapiOptions.connection);
 
-  getHoodieServer(hoodie_options, function (error, server, config) {
+  server.register({
+    register: hoodie,
```

Comment:
good catch! Funny enough it seems to work like this, too. Looks like `hapi` has an internal check for these cases, cool!